server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+const app = express()
+
+app.get('/ping',(req,res)=>{
+    res.send('Ping')
+})
+
+app.listen(3000,()=>{
```

Comment:
Consider adding error handling middleware to handle any potential errors that may occur during the execution of your server. This is a best practice and can help with debugging. [important]